File: programmers/Level1/가운데_글자_가져오기.py
def solution(s):
    print(s)
    if len(s) % 2 == 0:
        answer = s[int(len(s) / 2)-1 : int(len(s)/2)+1]
    else:
        answer = s[int(len(s)/2)]
    print(answer)
    return answer

File: programmers/Level1/가운데_글자_가져오기.cpp
#include <string>

using namespace std;

string solution(string s) {
	int len = s.length();
	if (len % 2 == 0)
		return s.substr((len-1) / 2, 2);
	return s.substr((len / 2), 1);
}

// substr(position, length);
